bootstrap/007_probes.sql

```diff
@@ -291,17 +291,19 @@ BEGIN
     USING (
         SELECT *
         from (values
-                ('Long Queries', 'start_time < dateadd(minute, -10, current_timestamp()) AND NOT QUERY_TYPE = \'EXECUTE_STREAMLIT\''),
-                ('Big Readers', 'bytes_scanned > 10000000000')
-             )) s (name, condition)
+                ('Long Queries', 'start_time < dateadd(minute, -10, current_timestamp()) AND NOT QUERY_TYPE = \'EXECUTE_STREAMLIT\'', False),
+                ('Big Readers', 'bytes_scanned > 10000000000', False),
+                ('Costs 10 Credits', 'tools.approx_credits_used(warehouse_name, start_time) > 10', True),
+                ('Costs 50 Credits', 'tools.approx_credits_used(warehouse_name, start_time) > 50', True)
```

Comment:
I think we would want these to be greater-than-or-equal-to instead of greater-than. `approx_credits_used` is going to return a NUMBER (integer), which would mean the queries we match are actually those which cost at least 11 or 51 credits

Reply:
NUMBER is ~DECIMAL so it will have a decimal.
![20230830-165039](https://github.com/sundeck-io/OpsCenter/assets/2022305/08182801-07c0-4487-85a1-884c2587b230)

Reply:
ah, I had just done `SELECT 10.0::NUMBER` and it truncated it. My sanity check was just invalid :)